libpysal/graph/_plotting.py

```diff
@@ -135,3 +136,36 @@ def _plot(
             ax.scatter(coords[:, 0], coords[:, 1], **node_kws, zorder=2)
 
     return ax
+
+
+def _explore_graph(g, gdf, **kwargs):
+    """Plot graph as an interactive Folium Map
+
+    Parameters
+    ----------
+    g : libpysal.Graph
+        graph to be plotted
+    gdf : geopandas.GeoDataFrame
+        geodataframe used to instantiate to Graph
+    kwargs: additional keyword arguments passed to the geopandas explore
+        method. For example, to make the  nodes larger pass `marker_kwds={'radius':8}`
+
+    Returns
+    -------
+    folium.Map
+        folium map
+    """
+    gdf = gdf.copy()
+    gdf['id']=gdf.index.values
+    m = gdf.reset_index()[['id', 'geometry']].set_geometry(gdf.centroid).explore(**kwargs)
+
+    adj = g.adjacency.reset_index()
+
+    origins = gdf.loc[adj.focal].centroid.geometry
+    destinations = gdf.loc[adj.neighbor].centroid.geometry
```

Comment:
```suggestion
    origins = gdf.loc[adj.focal].centroid
    destinations = gdf.loc[adj.neighbor].centroid
```
I know this may be obsolete but centroid is always a GeoSeries.